pytest.ini

```diff
@@ -0,0 +1,3 @@
+[pytest]
+filterwarnings =
+    ignore::DeprecationWarning:google
```

Comment:
This change looks unrelated to the pull request and should be removed.

Reply:
Done